### DISCUSSION TITLE
Assign a window to multiple tags and view multiple tags

### DISCUSSION THREAD

**Discussion by adamcstephens** [leftwm/leftwm#1085] [Ideas]
I couldn't find an issue or discussion about this. If they exist, feel free to point me to them and close.

I'm interested in assigning a window to multiple tags, but not all. So for example if I have 9 tags, a window could be assigned to 1 AND 2. I don't see a way to do this, with sticky windows being the only way I can find to view a window across tags. Is this currently possible, or would this be a feasible change?

Is it possible to view windows from multiple tags at once? e.g. I can see all the windows from tags 1 AND 2 at the same time.

**Comment by VuiMuich** [comment]
Our general concept of tags would definitely allow for such a feature, but currently this is not implemented. 
If someone would like to propose a PR for this feature it would be surly very much appreciated and welcomed.

**Comment by hertg** [comment]
@adamcstephens I had the same feature request in mind for a while, I opened up #1087 for this request. Be sure to give it a thumbs up if you like it and consider leaving feedback if you have anything to add :)
